### DISCUSSION TITLE
imtoken Dapp页面开发，无法展示加载的iframe页面

### DISCUSSION THREAD

**Discussion by wfyweb** [consenlabs/webview#55] [Q&A] [answered]
imtoken Dapp页面开发，无法展示加载的iframe页面，请问如何配置，能加载iframe页面

**Comment by xwartz** [comment, marked as answer]
imToken 应该没有限制 iframe，有具体事列吗

**Reply by wfyweb** [reply]
找到问题了， 是因为没有配置有效的https证书，如果用有效https加域名访问是没问题的。感谢解答！
